lightning/src/ln/peer_handler.rs

```diff
@@ -178,16 +178,12 @@ pub struct MessageHandler<CM: Deref, RM: Deref> where
 ///
 /// For efficiency, Clone should be relatively cheap for this type.
 ///
-/// If applicable in your language, you probably want to just extend an int and put a file
-/// descriptor in a struct and implement this trait on it. Note, of course, that if you do so and
-/// have multiple threads interacting with the [`PeerManager`], you must not call `close()` on the
-/// file descriptor until all threads are guaranteed to not call any [`PeerManager`] functions with
-/// the same descriptor. Otherwise, file descriptor re-use could imply that you call a
-/// [`PeerManager`] function with a file descriptor representing a different connection.
-///
-/// Note that if you are using a higher-level net library that may call close() itself when you
-/// close a socket, be careful to ensure you don't directly use the file descriptor for your [`Eq`]
-/// and [`Hash`] implementations.
+/// Two descriptors may compare equal (by [`cmp::Eq`] and [`hash::Hash`]) as long as the original
+/// has been disconnected, the [`PeerManager`] informed of the disconnection (either by it having
```

Comment:
was informed

Reply:
I think its correct? "as long as the original has been disconnected, the PeerManager informed of the disconnection (...), and no further calls related to the original socket occur."?

Reply:
Seem like an auxiliary verb such as "is", "was", or "has been" is necessary before "informed" given a new subject is introduced in the second item of the list.